source/blog/032-embedded-sass-is-live.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "Embedded Sass is Live"
+author: Natalie Weizenbaum
+date: 2022-02-01 2:00 PST
+---
+
+After several years of planning and development, I'm excited to finally announce
+the stable release of Embedded Dart Sass along with its first official wrapper,
+the [`sass-embedded`] package available now on npm!
+
+[`sass-embedded`]: https://www.npmjs.com/package/sass-embedded
+
+Embedded Sass is an ongoing effort to make a highly-performant Sass library
+available to as many different languages as possible, starting with Node.js.
+Although Node.js already has access to the pure-JS `sass` package, the nature of
+JavaScript inherently limits how quickly this package can process large Sass
+files especially in asynchronous mode. We expect `sass-embedded` to be a major
+boon to developers for whom compilation speed is a concern.
+
+The `sass-embedded` package fully supports the [new JS API] as well as the
+[legacy API] other than a few cosmetic options. You can use it as a drop-in
+replacement for the `sass` package, and it should work with all the same build
+plugins and libraries. Note the `sass` package is much faster at compiling files
+synchronously, the `sass-embedded` package is actually a bit faster at compiling
+them *asynchronously* instead—although for substantial Sass files it will
+generally be much faster than `sass` either way.
```

Comment:
Since there are two axes of comparison, this is a bit confusing.

```suggestion
plugins and libraries. Note that `sass-embedded` is a bit faster in
*asynchronous* mode than it is in synchronous mode (whereas the `sass`
package was faster in synchronous mode). For substantial Sass files, running
`sass-embedded` in either mode will generally be much faster than `sass`.
```